modules/virt-creating-udn-namespace-cli.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc              
+
+:_mod-docs-content-type: PROCEDURE                                    
+[id="virt-creating-udn-namespace-cli_{context}"]                                   
+= Creating a namespace for user-defined networks by using the CLI
+
+You can create a namespace to be used with primary user-defined networks (UDNs) by using the CLI.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+
+
+.Procedure
+. Create a `Namespace` object by running the following command:
```

Comment:
This step does not include running a command. I see steps similar to this in the docs.

```suggestion
. Create a `Namespace` object as a YAML file similar to the following example:
```